.github/workflows/codeql.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     # Autobuild attempts to build any compiled languages (C/C++, C#, Go, Java, or Swift).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@3c3833e0f8c1c83d449a7478aa59c036a9165498 #v3
+      uses: github/codeql-action/autobuild@2d92b76c45b91eb80fc44c74ce3fce0ee94e8f9d #v3
```

Comment:
Ah, it didn't update the comment. If the comment was the full version, then it would be updated.

Personally, I prefer the full version in the comment so I don't have to look up the commit to find the exact version, when troubleshooting unexpected behavior after updates.